### PR TITLE
sharpen up contribution agreement section

### PR DESCRIPTION
**Pull request by stuij**
note that contributors should add themselves as copyright owners

### CODE REVIEW

**Comment by stuij** [comment]
For the licensing changes I'll actually make a toplevel LICENSE file which will detail our license situation.

**Review by kbeyls — APPROVED** [review]
LGTM - thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,3 +135,9 @@ under the same terms as those in the [LICENSE](LICENSE.md) file.
 
 We _do not_ require copyright assignment. The original contributor
 will retain the copyright.
+
+When adding content for the first time to an existing ABI specification or when
```
Comment (sallyarmneale) [inline CONTRIBUTING.md:139]: you add

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -135,3 +135,9 @@ under the same terms as those in the [LICENSE](LICENSE.md) file.
...
+creating a new one, you should add the copyright owner (presumably either
```
Comment (sallyarmneale) [inline CONTRIBUTING.md:140]: you create

**Pull request merged**